test/main/store/actions/index.test.js

```diff
@@ -980,3 +983,66 @@ describe('#setBlockHeight', () => {
     expect(main.networksMeta.ethereum).toStrictEqual({ 1: { blockHeight: 0 }, 4: { blockHeight: 500 }, 137: { blockHeight: 0 } })
   })
 })
+
+describe('#removeBalances', () => {
+  let balances
+
+  const updaterFn = (node, address, update) => {
+    expect(node).toBe('main.balances')
+    expect(address).toBe(owner)
+
+    balances = update(balances)
+  }
+
+  const removeBalances = setToRemove => removeBalancesBySetAction(updaterFn, owner, setToRemove)
+
+  beforeEach(() => {
+    balances = Object.values(testTokens).map(token => ({...token, balance: addHexPrefix(new BigNumber(120).toString(16))}))
+  })
+
+  it('should remove all tokens from the removal set from an accounts balance', () => {
```

Comment:
might want to add a test to make sure it removes the balance from _all_ the accounts that contain it.

Reply:
nm this one, see that this function only operates on one account now :+1: